docs/reference/query-dsl/multi-match-query.asciidoc

```diff
@@ -516,3 +519,55 @@ per-term `blended` queries. It accepts:
 ===================================================
 The `fuzziness` parameter cannot be used with the `cross_fields` type.
 ===================================================
+
+[[type-bool-prefix]]
+==== `bool_prefix`
+
+The `bool_prefix` type's scoring behaves like <<type-best-fields>>, but using a
```

Comment:
This is not correct, we use the `most-fields` mode for this query so we sum all field scores like a normal disjunction. The `tie_breaker` option can be used to change this behavior so I wonder if we should rename `best_fields` and `most_fields` to `bool` and rely on this option solely. That's not in the scope of this pr though so let's fix this new docs for now.

Reply:
Thanks for catching this, I see what my misunderstanding about the code here was